server/server.js

```diff
@@ -54,8 +55,24 @@ app.use(methodOverride());
 // LoopBack REST interface
 app.use(app.get('restApiRoot'), loopback.rest());
 
-// API explorer
-require('loopback-component-explorer')(app);
+// To prevent the snyk warning about swagger-ui@2.x, the explorer component
+// is moved from dependencies to devDependencies.
+// Here we try to mount the explorer component in case it's not installed.
+
+try {
+  require('loopback-component-explorer')(app);
+} catch (e) {
+  const notFound = e.code === 'MODULE_NOT_FOUND' && e.message;
+  if (notFound) {
+    debug(
+      'loopback-workspace runs without API Explorer enabled.' +
+      'To install it, run `npm i loopback-component-explorer`.',
+    );
+  } else {
+    debug('Cannot load loopback-component-explorer: %s', e.stack || e);
+  }
+}
+
 app.once('started', function(baseUrl) {
   g.log('Browse your REST API at %s%s', baseUrl, '/explorer');
```

Comment:
This log should not be printed if Explorer is not enabled.

Reply:
good catch.